plugins/iframe-resizer/host.js

```diff
@@ -0,0 +1,30 @@
+export function hostResizer(host) {
+	let initialized = false;
+	host.onEvent('ready', () => {
+		if (initialized) {
+			return;
+		}
+
+		const resize = () => {
+			window.iFrameResize(
+				{
+					log: host.debugEnabled,
+					heightCalculationMethod: 'max'
+				},
+				host.iframe
+			);
+		};
+
+		const resizerScript = document.createElement('script');
+		resizerScript.src = 'https://s.brightspace.com/lib/iframe-resizer/3.6.5/iframeResizer.min.js';
```

Comment:
`ifrau-resizer` is only available as a CJS package, so we can't directly import it anymore. Instead, we can push it to the CDN and include it in the host page via a script tag. Thanks for the help with the CDN pieces, @dlockhart!